sw/device/tests/BUILD

```diff
@@ -1678,3 +1678,23 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
```

Comment:
likewise you can run `bazel run //quality:buildifier_fix` to format this file according to CI standards.

Reply:
Will run it for the final PR.